main.js

```diff
@@ -0,0 +1,55 @@
+var board = [
+            ["1"],
+            ["2"],
+            ["5", "4", "3"]
+            ];
+
+var boardPrint = board.map(function(peg) {
+  return peg.map(function(disc) {
+    return disc;
+  }).join(' ');
+});
+var display = boardPrint.join('\n');
+
+var moveDisc = function (a, b) {
+
+if (board[a-1][board[a-1].length-1] < board[b-1][board[b-1].length-1] || board[b-1][board[b-1].length-1] === undefined) {
+  let removed = board[a-1].pop();
+  board[b-1].push(removed);    
+
+  var boardPrint = board.map(function(peg) {
```

Comment:
Again, wrapping this in a function would prevent you from having to copy/pasting the code